src/app/cluster/cluster-details/alertmanager-config/component.ts

```diff
@@ -9,37 +9,47 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-import {Component, Input, OnDestroy} from '@angular/core';
+import {Component, Input, OnDestroy, OnInit} from '@angular/core';
 import {MatDialog, MatDialogConfig} from '@angular/material/dialog';
 import {NotificationService} from '@core/services/notification';
 import {MLAService} from '@core/services/mla';
+import {SettingsService} from '@core/services/settings';
 import {ConfirmationDialogComponent} from '@shared/components/confirmation-dialog/component';
 import {Cluster} from '@shared/entity/cluster';
 import {AlertmanagerConfig} from '@shared/entity/mla';
+import {AdminSettings} from '@shared/entity/settings';
 import * as _ from 'lodash';
 import {Subject} from 'rxjs';
-import {filter, switchMap, take} from 'rxjs/operators';
+import {filter, switchMap, take, takeUntil} from 'rxjs/operators';
 import {AlertmanagerConfigDialog} from './alertmanager-config-dialog/component';
 
 @Component({
   selector: 'km-alertmanager-config',
   templateUrl: './template.html',
   styleUrls: ['./style.scss'],
 })
-export class AlertmanagerConfigComponent implements OnDestroy {
+export class AlertmanagerConfigComponent implements OnInit, OnDestroy {
   @Input() cluster: Cluster;
   @Input() projectID: string;
   @Input() isClusterRunning: boolean;
   @Input() alertmanagerConfig: AlertmanagerConfig;
 
+  private _settings: AdminSettings;
   private readonly _unsubscribe = new Subject<void>();
 
   constructor(
     private readonly _matDialog: MatDialog,
     private readonly _mlaService: MLAService,
-    private readonly _notificationService: NotificationService
+    private readonly _notificationService: NotificationService,
+    private readonly _settingsService: SettingsService
   ) {}
 
+  ngOnInit(): void {
+    this._settingsService.adminSettings.pipe(takeUntil(this._unsubscribe)).subscribe(settings => {
+      this._settings = settings;
```

Comment:
nit: In case of the one liners you can avoid using `{}`.

Reply:
done :)